### PR TITLE
Crypto samples change logging backend to UART

### PR DESCRIPTION
**Pull request by mswarowsky**
There is no reason to use RTT for the crypto samples, so switch to RTT to make the config simpler

### CODE REVIEW

**Review by SebastianBoe — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

File: samples/crypto/persistent_key_usage/prj.conf [hunk 1]
```diff
@@ -4,14 +4,9 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 
-# Enable loging using RTT and UART
+# Enable loging
```
Comment (frkv) [inline samples/crypto/persistent_key_usage/prj.conf:7]: Typo: loging => logging (multiple files)

File: samples/crypto/aes_cbc/prj.conf [hunk 2]
```diff
@@ -4,14 +4,9 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 
-# Enable loging using RTT and UART
+# Enable loging
```
Comment (torsteingrindvik) [inline samples/crypto/aes_cbc/prj.conf:7]: logging*

File: samples/crypto/persistent_key_usage/prj.conf (hunk 1 above)
Comment (mswarowsky) [reply samples/crypto/persistent_key_usage/prj.conf:7]: Wonder how often that typo already got copy pasted, as it is in all samples :D, gonna fix it

**Review by frkv — CHANGES_REQUESTED** [review]
Documentation for all crypto samples states output to RTT. If this change is to happen, then documentatin must reflect this

**Review by frkv — APPROVED** [review]
LGTM

**Review by mia-ko — CHANGES_REQUESTED** [review]
Doc changes look ok to me :)
Two questions:
1. Does this also apply to crypto tests? I see they are still mentioning RTT.
2. Should we have an entry in the changelog for this?

**Comment by melwee01** [comment]
Is this logging integrated with the logging levels we are putting in with https://github.com/nrfconnect/sdk-nrf/pull/10266/files?

**Review by mia-ko — APPROVED** [review]

**Comment by mswarowsky** [comment]
> Is this logging integrated with the logging levels we are putting in with https://github.com/nrfconnect/sdk-nrf/pull/10266/files?

No, that doesn't seem to be related

**Pull request merged**